apps/emqx_management/src/emqx_mgmt_api_app.erl

```diff
@@ -143,26 +147,30 @@ api_key(post, #{body := App}) ->
     Desc = unicode:characters_to_binary(Desc0, unicode),
     case emqx_mgmt_auth:create(Name, Enable, ExpiredAt, Desc) of
         {ok, NewApp} -> {200, format(NewApp)};
-        {error, Reason} -> {400, io_lib:format("~p", [Reason])}
+        {error, Reason} ->
+            {400, #{code => 'BAD_REQUEST',
+                message => iolist_to_binary(io_lib:format("~p", [Reason]))}}
     end.
 
+-define(NOT_FOUND_RESPONSE, #{code => 'NOT_FOUND', message => <<"Name NOT FOUND">>}).
+
 api_key_by_name(get, #{bindings := #{name := Name}}) ->
     case emqx_mgmt_auth:read(Name) of
         {ok, App} -> {200, format(App)};
-        {error, not_found} -> {404, <<"NOT_FOUND">>}
+        {error, not_found} -> {404, ?NOT_FOUND_RESPONSE}
     end;
 api_key_by_name(delete, #{bindings := #{name := Name}}) ->
     case emqx_mgmt_auth:delete(Name) of
         {ok, _} -> {204};
-        {error, not_found} -> {404, <<"NOT_FOUND">>}
+        {error, not_found} -> {404, ?NOT_FOUND_RESPONSE}
     end;
 api_key_by_name(put, #{bindings := #{name := Name}, body := Body}) ->
     Enable = maps:get(<<"enable">>, Body, undefined),
     ExpiredAt = ensure_expired_at(Body),
     Desc = maps:get(<<"desc">>, Body, undefined),
     case emqx_mgmt_auth:update(Name, Enable, ExpiredAt, Desc) of
         {ok, App} -> {200, format(App)};
-        {error, not_found} -> {404, <<"NOT_FOUND">>}
+        {error, not_found} -> {404, ?NOT_FOUND_RESPONSE}
```

Comment:
Seems the macro have not defined

Reply:
line 155 ：0

Reply:
Yep. Maybe we should move it to front of the module?